vendor/github.com/Cray-HPE/shasta-cfg/customizations.yaml

```diff
@@ -732,6 +737,8 @@ spec:
         ingresses:
           ingressgateway:
             issuers:
+              shasta-cmn: https://api.cmn.{{ network.dns.external }}/keycloak/realms/shasta
+              keycloak-cmn: https://auth.cmn.{{ network.dns.external }}/keycloak/realms/shasta
```

Comment:
Looks like this may be another one.   These should not be added back.   Looks like they were removed directly in the csm repo.